genie-core/src/test/java/com/netflix/genie/core/configs/ServicesConfigTest.java

```diff
@@ -263,8 +263,7 @@ public JobSubmitterService jobSubmitterService(
         final ApplicationEventPublisher eventPublisher,
         final ApplicationEventMulticaster eventMulticaster,
         final List<WorkflowTask> workflowTasks,
-        @Qualifier("jobsDir")
-        final Resource genieWorkingDir,
+        @Qualifier("jobsDir") final Resource genieWorkingDir,
```

Comment:
Ok this is _my_ personal OCD, but this this line change (and a couple others) is unrelated to this feature/patch.
Sneaking it in can make it harder to track down changes and their reason via `git blame`.
Beside, isn't it generally recommended to have each annotation on a new line?
e.g.: 
>[each annotation is listed on a line of its own (that is, one annotation per line)](https://google.github.io/styleguide/javaguide.html#s4.8.5-annotations)

> [When used on a declaration, each annotation often appears, by convention, on its own line](https://docs.oracle.com/javase/tutorial/java/annotations/basics.htm)